.github/CODEOWNERS

```diff
@@ -782,15 +782,38 @@ packages/kbn-yarn-lock-validator @elastic/kibana-operations
 # used for the 'team' designator within Kibana Stats
 
 # Data Discovery
-/x-pack/test/functional/apps/discover/ @elastic/kibana-data-discovery
-/x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/discover/ @elastic/kibana-data-discovery
-/test/functional/apps/discover/ @elastic/kibana-data-discovery
-/test/functional/apps/context/ @elastic/kibana-data-discovery
-/test/api_integration/apis/unified_field_list/ @elastic/kibana-data-discovery
-/x-pack/test/search_sessions_integration/ @elastic/kibana-data-discovery
-/test/plugin_functional/test_suites/data_plugin @elastic/kibana-data-discovery
-/examples/demo_search/ @elastic/kibana-data-discovery
+/examples/demo_search @elastic/kibana-data-discovery
+/test/accessibility/apps/discover.ts @elastic/kibana-data-discovery
+/test/accessibility/apps/management.ts @elastic/kibana-data-discovery
```

Comment:
rename and take ownership

Reply:
Oops, looks like I had this one wrong. `/test/accessibility/apps/management.ts` only includes Index Management tests, not Data Views Management tests. I've dropped this line instead since Index Management belongs to Platform Deployment Management.